galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/utils/GalasaGsonBuilder.java

```diff
@@ -13,7 +13,7 @@
 public class GalasaGsonBuilder {
 
     public static Gson build() {
-        GsonBuilder builder = new GsonBuilder();
+        GsonBuilder builder = new GsonBuilder().disableHtmlEscaping();
```

Comment:
I think we need to explain why we have to disable html escaping. It's not clear why we need to do this.